index.html

```diff
@@ -258,83 +228,94 @@ <h3>Credential Schema</h3>
-                  "name": "Bachelor of Science and Arts"
-                  "type": "Bachelor's Degree",
-                },
-                "issuingInstitution": "Very Good University"
-              },
-              "credentialSchema": {
-                "id": "https://example.org/examples/degree.json",
-                "type": "CredentialSchema2022"
-              }
-            }
-            </pre>
-          </p>
-          <p>
-            <b>Credential Schema</b>
-            <pre class="example highlight" title="Example University Degree Credential Schema">
-            {
-              "id": "06e126d1-fa44-4882-a243-1e326fbe21db",
-              "type": "https://w3c.github.io/vc-json-schema/",
-              "name": "UniversityDegreeCredential",
-              "issuer": "did:example:MDP8AsFhHzhwUvGNuYkX7T",
-              "validFrom": "2021-01-01T00:00:00+00:00",
-              "schema": {
-                "$schema": "https://json-schema.org/draft/2020-12/schema",
-                "description": "Degree Schema",
-                "type": "object",
-                "properties": {
-                  "credentialSubject": {
-                    "type": "object",
-                    "properties": {
-                      "degree": {
-                        "type": "object",
-                        "properties": {
-                          "name": {
-                            "type": "string"
-                          },
-                          "type": {
-                            "type": "string"
-                          }
-                        },
-                        "required": ["name", "type"]
+            <h3>CredentialSchema2023</h3>
+            <p>
+               This <a href="https://www.w3.org/TR/json-ld/#dfn-term-definition">term definition</a>
+               is <code>https://www.w3.org/ns/credentials/issuer-dependent#CredentialSchema2023</code>.
+            </p>
+            <p>
+               <b>CredentialSchema2023</b> is for the validation of W3C Verifiable Credentials, based
+               on representing JSON Schema in a <a>verifiable credential</a>. The version of [[JSON-Schema]]
+               can be any version noted in the section on <a href="#json-schema-specifications">JSON Schema Specifications</a>.
+            </p>
+            <p>
+             <table class="simple">
+                <thead>
+                   <tr>
+                      <th>Property</th>
+                      <th>Description</th>
+                   </tr>
+                </thead>
+                <tbody>
+                   <tr>
+                      <td>id</td>
+                      <td>The constraints on the <code>id</code> property are listed in the Verifiable Credentials
+                      Data Model specification [[VC-DATA-MODEL]]. The value is expected to be a URL that identifies
+                      the credential schema associated with the <a>verifiable credential</a>.</td>
```

Comment:
I think it's important to clarify how implementers of a verifier go from "id" to the actual json_schema object that will be used to validate against.

Reply:
there is a section on resolution and #19 to handle this